src/ad-cid.js

```diff
@@ -29,13 +29,28 @@ export function getAdCid(adElement) {
   if (!config || !config.clientIdScope) {
     return Promise.resolve();
   }
-  const cidPromise = cidForDocOrNull(adElement.getAmpDoc()).then(cidService => {
+  return getClientScopedAdCid(adElement.getAmpDoc(), adElement.win,
+                              config.clientIdScope, config.clientIdCookieName);
+}
+
+/**
+ * @param {!./service/ampdoc-impl.AmpDoc} doc
+ * @param {!Window} win
+ * @param {!string} clientIdScope
+ * @param {string=} opt_clientIdCookieName
+ * @return {!Promise<string|undefined>} A promise for a CID or undefined if
+ *     - the ad network does not request one or
+ *     - `amp-analytics` which provides the CID service was not installed.
+ */
+export function getClientScopedAdCid(
```

Comment:
`ClientScopedAdCid` does not make sense, since CID implies "client", and they're always scoped.